tests/negative/security/tls.go

```diff
@@ -53,6 +61,16 @@ MIGkAgEBBDB6yW6RIYfTXdYVuPY0V0L6EtZ6vZD86vgbsw52Y3/U5nZ2JE++JrKu
 tt2Xt/NMzG6gBwYFK4EEACKhZANiAAQDEhfHEulYKlANw9eR5l455gwzAIQuraa0
 49RhvM7PPywaiD8DobteQmE8wn7cJSzOYw6GLvrL4Q1BO5EFUXknkW50t8lfnUeH
 veCNsqvm82F1NVevVoExAUhDYmMREa4=
+-----END EC PRIVATE KEY-----`)
+
+	privateKey1 = []byte(`-----BEGIN EC PARAMETERS-----
```

Comment:
Can you drop a comment here with the details on how the private key was generated?  Perhaps even the `openssl` command?